src/managing-deis/platform-ssl.md

```diff
@@ -1,3 +1,58 @@
 # Platform SSL
 
-TODO (bacongobbler): rewrite for v2
+SSL/TLS is the standard security technology for establishing an encrypted link between a web server
+and a browser. This link ensures that all data passed between the web server and browsers remain
+private and integral.
+
+To enable SSL for your cluster and all apps running upon it, you can add an SSL key to your load
+balancer. You must either provide an SSL certificate that was registered with a CA or provide
+[your own self-signed SSL certificate](../reference-guide/creating_self_signed_ssl.md).
+
+## Installing SSL on a Load Balancer
+
+On most cloud-based load balancers, you can install a SSL certificate onto the load balancer
+itself. Any communication inbound to the cluster will be encrypted while the internal components
+of Deis will still communicate over HTTP.
+
+To enable SSL, you will need to open port 443 on the load balancer and forward it to port 80 on the
+routers. For AWS, you'll also need to add port 443 in the security group settings for your load
+balancer.
+
```

Comment:
If doing it this way (using node ports) you'd need a health-check in place as well so that the load balancer only sends traffic to nodes that are actually running a router instance.  Otherwise, the high-numbered ports allocated dynamically by k8s should be used on the back end... unless these directions are strictly for _modifying_ the load balancer to support SSL and pre-suppose that the load balancer was set up correctly to begin with.